doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -373,7 +373,7 @@ Creating Hints
      The default value for hint locality will change in a future
      release. Hints added outside of sections without an explicit
      locality are now deprecated. We recommend using :attr:`export`
-     where possible.
+     where possible. This warning will raise an error.
```

Comment:
You should add a "by default" somewhere to this sentence, otherwise the wording is weird. Same for the other instances.

Reply:
Sure this is "by default", but can the user even change the fatality? Other than disabling the warning I don't see how it would be relevant.

Reply:
You can set the warning back to its "warn" behaviour, or silence it using the Warnings command.

Reply:
OK will update